.semaphore/semaphore.yml

```diff
@@ -115,8 +117,10 @@ blocks:
 
   - name: Run first block of tests (mostly Kafka and KStreams)
     dependencies: ["☕️ Build and Test Java (Kafka and Kafka Streams) Only"]
+    run:
+      when: "(branch = 'master' OR branch = 'release') OR change_in(['../_includes/tutorials/**/kstreams/**/*', '../_includes/tutorials/**/kafka/**/*', '../_includes/tutorials/**/scala/**/*', '../_data/harnesses/**/kstreams.yml', '../_data/harnesses/**/kafka.yml'])"
     execution_time_limit:
-      minutes: 20
+      minutes: 10
```

Comment:
Reducing the timeout for builds to 10 minutes should be long enough.  We still have flaky tests that won't pass hence holding up the build artificially.